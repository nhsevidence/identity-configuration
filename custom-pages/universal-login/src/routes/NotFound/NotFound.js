import React from 'react'
import { createBrowserHistory } from 'history'
import './NotFound.scss'

const goBack = (e) => {
  e.preventDefault()
  return createBrowserHistory.goBack()
}

export const NotFound = () => (
  <div className="notFound">
    <h4>Something must have gone slightly wrong!</h4>
    <p><a href="#" onClick={goBack}>&larr; Back</a></p>
  </div>
)

export default NotFound
