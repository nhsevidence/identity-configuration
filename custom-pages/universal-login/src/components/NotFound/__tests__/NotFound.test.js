import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import NotFound from '../NotFound'

Enzyme.configure({ adapter: new Adapter() })

describe('NotFound components', () => {
  it('should render NotFound correctly', () => {
    expect(shallow(<NotFound />)).toMatchSnapshot()
  })
})
