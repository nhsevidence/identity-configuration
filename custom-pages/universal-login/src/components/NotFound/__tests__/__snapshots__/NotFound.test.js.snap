// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound components should render NotFound correctly 1`] = `ShallowWrapper {}`;
